packages/docs/next.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   basePath: isProduction ? URL_PREFIX : '',
   output: 'export',
   env: {
-    CODE_SANDBOX_URL: `https://codesandbox.io/s/github/bigcommerce/big-design/tree/%40bigcommerce/examples%40${EXAMPLES_VERSION}/packages/examples`,
+    CODE_SANDBOX_URL: `https://codesandbox.io/p/devbox/github/bigcommerce/big-design/tree/%40bigcommerce/examples%40${EXAMPLES_VERSION}/packages/examples`,
```

Comment:
Do we still need to point to version tag here?

Reply:
It still would be useful for different versions of the examples, e.g. we decide to change some stuff up.